tests/test_pac_bio_experiment.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+from ml_warehouse.schema import PacBioRun
+from sqlalchemy import select
+
+from lang_qc.models.pacbio.experiment import PacBioExperiment
+from tests.conftest import insert_from_yaml
+
+
+def test_query(mlwhdb_test_session):
+
+    run_name = "TRACTION-RUN-92"
+    # Four wells, D1 has 40 samples, the rest have one sample each.
+
+    insert_from_yaml(
```

Comment:
This could be a reusable fixture in its own right. Unless you want to query before and after data loads.

Reply:
This is true. Also this could have absorbed some other standard fixtures (loading dict data to langqc db). However, fixtures with arguments are written a bit fifferently. Let's have separate pr soon to create this new standard fixture